packages/components/src/FormField/FormField.tsx

```diff
@@ -1,11 +1,13 @@
 import React, { useId } from "react";
+import classNames from "classnames";
 import { FormFieldProps } from "./FormFieldTypes";
 import { FormFieldWrapper } from "./FormFieldWrapper";
 import { FormFieldPostFix } from "./FormFieldPostFix";
 import { useAtlantisFormFieldActions } from "./hooks/useAtlantisFormFieldActions";
 import { useAtlantisFormField } from "./hooks/useAtlantisFormField";
 import { useAtlantisFormFieldName } from "./hooks/useAtlantisFormFieldName";
 import { useAtlantisReactHookForm } from "./hooks/useAtlantisReactHookForm";
+import styles from "./FormField.module.css";
```

Comment:
don't love that I'm pulling styles in here. I feel like an effort was made to separate it.

the alternative is to pass both `value` and `type` into the hook where we build `textFieldProps` so that we can build the `textFieldProps.className` with the logic